vnfs/libexec/wwmkchroot/include-ubuntu

```diff
@@ -2,7 +2,8 @@
 APT_CONF="root/ww-apt.conf"
 DEB_SRCS="root/sources.list"
 DEB_ISOS="root/iso.list"
-APT_CMD="apt-get -c $CHROOTDIR/$APT_CONF -o Dir::Etc::SourceList=/$CHROOTDIR/"
+#APT_CMD="env DEBIAN_FRONTEND="noninteractive" apt-get -c $CHROOTDIR/$APT_CONF -o Dir::Etc::SourceList=/$CHROOTDIR/"
```

Comment:
Can you remove the commented out lines here and below? They'll be in git's history so no need to keep them preserved in-situ.